install.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
**nitpick** Would it make sense to throw this in a `./scripts` directory? Might not be worth it if it's short lived but could help declutter the root.